src/pyj/read_book/paged_mode.pyj

```diff
@@ -661,19 +662,24 @@ def handle_gesture(gesture):
                 get_boss().send_message('next_section', forward=gesture.direction is 'up')
         else:
             if not gesture.active or gesture.is_held:
-                scroll_by_page(gesture.direction is 'right', True)
+                scroll_by_page(gesture.direction is 'right', True, flip_if_rtl_page_progression=True)
+    # Gesture progression direction is determined in the gesture code;
+    # don't set flip_if_rtl_page_progression=True here.
     elif gesture.type is 'prev-page':
```

Comment:
I pondered changing the gesture code to send "go left" and "go right" actions, but it seemed pointless because the gesture code has to know what the page progression direction is in order to make the left or right side a bigger touch region.